src/Stratis.Bitcoin.Features.Consensus/ConsensusLoop.cs

```diff
@@ -494,11 +469,11 @@ public void ValidateBlock(RuleContext context, bool skipRules = false)
         /// Validates a block using the consensus rules and executes it (processes it and adds it as a tip to consensus).
         /// </summary>
         /// <param name="context">A context that contains all information required to validate the block.</param>
-        internal async Task ValidateAndExecuteBlockAsync(RuleContext context)
+        internal async Task ValidateAndExecuteBlockAsync(RuleContext context, bool skipRules = false)
```

Comment:
In the previous iteration, the default skipRules was `true` but now it's `false`.
Is that intentional?

Reply:
Yes its now derived from the caller and its still true when we come from the method `ValidateAndExecuteBlockAsync`